### PR TITLE
fix(editor): Show only error title and 'Open errored node' button; hide 'Ask Assistant' in root for sub-node errors

### PR DESCRIPTION
**Pull request by igatanasov**
## Summary

Show only error title and 'Open errored node' button; hide 'Ask Assistant' in root for sub-node errors
## Related Linear tickets, Github issues, and Community forum posts

[
](https://linear.app/n8n/issue/ADO-2730/feature-change-error-display-of-sub-node)

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/Error/NodeErrorView.vue [hunk 1]
```diff
@@ -132,6 +132,13 @@ const assistantAlreadyAsked = computed(() => {
 	});
 });
 
+const isSubNodeError = computed(() => {
```
Comment (mutdmour) [inline packages/editor-ui/src/components/Error/NodeErrorView.vue:135]: can you please add some tests?

File: packages/editor-ui/src/components/Error/NodeErrorView.vue [hunk 2]
```diff
@@ -696,9 +702,14 @@ async function onAskAssistantClick() {
 		}
 	}
 
-	&__assistant-button {
+	&__button {
 		margin-left: var(--spacing-s);
 		margin-bottom: var(--spacing-xs);
+		flex-direction: row-reverse;
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/components/Error/NodeErrorView.vue:708]: I guess this is ok for now if this is the only instance where we need show an icon on the left. If we end in more situations like this I think it's better to update the `Button` component so it has slots for icons on both sides.

**Review by MiloradFilipovic — APPROVED** [review]
Works as expected. Left a minor comment for the code but good to go without it.

**Comment by janober** [comment]
Got released with [`n8n@1.68.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.68.0)

**Pull request merged**